### PR TITLE
Use simple log format as default for make run

### PR DESCRIPTION
**Pull request by somaritane**
This PR sets log format to simple for local make run target.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,17 +1,17 @@
 # Copyright 2021 Absa Group Limited
-# 
+#
```
Comment (kuritka) [inline Makefile:2]: The PR looks LGTM for me, but we must do any license changes in golic/config.yaml. 

The problem is, when license string differs from template, it injects new license. Here, we will have two licenses in makefile as a result. 

Could you please, in the scope of this PR, skip license change?

**Review by kuritka — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (somaritane) [reply Makefile:2]: @kuritka hmm, might be my editor acting and auto-removing extra spaces, and diff ignores this change. will surely amend

Comment (somaritane) [reply Makefile]: @kuritka amended

**Review by kuritka — APPROVED** [review]
👍

**Pull request merged**